core/sql/regress/compGeneral/EXPECTED006.SB

```diff
@@ -1686,9 +1686,10 @@ d3d9446802a44259755d38e6d163e820   852952723  b1d5781111d84f7b3fe45a0852e59758cd
 --- 1 row(s) selected.
 >>select sha2('the original data', 128) from dual;
 
-*** ERROR[8428] The argument to function SHA2 is not valid.
+*** ERROR[4045] The operand of function SHA2 must be numeric.
```

Comment:
The error is a bit misleading, since 128 is numeric. Ideally, the error message should say that we expect 224, 256, 384, or 512.